classes/__tests__/vehicle.test.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const VehicleConstructor = require('../vehicle-constructor.js');
+const {Car, Motorcycle } = require('../vehicle-class.js');
+
```

Comment:
for line 4 if you want you can do the same thing as line 3, but for class. (this is a suggestion and don't have to take it)